lib/Backends/BackendUtils.cpp

```diff
@@ -150,6 +162,9 @@ runtime::RuntimeBundle::create(const IRFunction &F,
     symbol.offset = addr;
     symbol.size = numBytes;
     symbol.type = *w->getType();
+    bool isOutput = checkIfOutput(v);
+    symbol.output = isOutput;
+    symbol.input = !isOutput;
```

Comment:
Question: Can it be that the same symbol is used as input and as output at the same time?

Reply:
I'm glad you asked, I'm not sure, I think for quantization it can be? But I believe for a single function a given symbol is one or the other. If there is a chance that input and output can both be true I can change this PR to properly set output, and leave input as true for now.

Reply:
Should we add a verifier check for this properly that a symbol is always an input or an output in a function, but not both at the same time?